cfme/tests/storage/test_volume_backup.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import pytest
+import random
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils import version
+
+
+pytestmark = [pytest.mark.ignore_stream("upstream"),
+              pytest.mark.usefixtures('setup_provider'),
+              pytest.mark.provider([OpenStackProvider], scope='module')]
+
+
+@pytest.mark.tier(3)
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
+def test_storage_volume_backup_edit_tag_from_detail(appliance, provider):
+    collection = appliance.collections.volume_backups.filter({'provider': provider})
+    backups = collection.all()
+    backup = random.choice(backups)
```

Comment:
**REQUIRED:** This assumes that a backup is already present on the provider. Are these a constant thing, or should we have code that _adds_ a backup. Could make more sense that we also would be checking inventory at the same time. My fear is that this will just fail if there are none available.

Reply:
@psav pointed right, I mention that fear in PR introduction only. I have a card for this RHCFQE-5459. but going add it in this PR only.